packages/api/src/learning-materials/queries/what-is-user-name.query.ts

```diff
@@ -0,0 +1,9 @@
+import { Query } from '@nestjs-architects/typed-cqrs';
+
+import type { UserId } from '../../users/users.types';
+
+export class WhatIsUserNameQuery extends Query<string> {
```

Comment:
It should be in users module. 
It's dependency on this module, like subscribing for event from different module. 
I've just noticed is not exactly the same thing like ports & adapters, because there is no dependency inversion :D 
New questions (queries) for another modules will cause changes in module which have an answer :/ 
It twisted me before. Sorry :P Now I see Ports & Adapters may be better, 
but I think we can live with such solution too.